tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -863,6 +863,49 @@ def forward_xp(self, inputs, xp):
         return tuple(b)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('shape,indices_or_sections', [
+    ((2,), []),
+    ((2, 4), 2),
+    ((2, 4), [2, -3]),
+    ((2, 4, 6), [2, 8]),
+    ((2, 4, 8), [2, 5]),
+    ((2, 4, 5), [1, -3]),
+    ((2, 4, 10), [1, 4]),
+    ((2, 4, 6), [4, 2, 1]),
+    ((2, 4, 6), [1, 3, -2]),
+    ((2, 4, 6), numpy.array([1, 2])),  # indices with 1-d numpy array
+    ((2, 4, 6), numpy.array([2])),  # indices with (1,)-shape numpy array
+    ((2, 4, 8), numpy.array(2)),  # sections numpy scalar
+    ((2, 4, 6, 8), numpy.array(2.0)),  # sections with numpy scalar, float
+    ((2, 4, 6, 8), 2.0),  # float type sections, without fraction
+    # indices with empty numpy indices
+    ((2, 4, 8, 10), numpy.array([], numpy.int32)),
+    ((2, 4, 5, 10), numpy.array([], numpy.float64)),
+])
```

Comment:
Could you swap the length of the 2nd and 3rd axes axes since these parameters were written for dsplit?